package.json

```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "methods": "1.1.2",
+    "spdy": "^4.0.2",
```

Comment:
Shouldn't this go in dev dependencies instead?

Reply:
We can do without `spdy` since Node.js support http2 natively now.